c2corg_ui/static/partials/feed.html

```diff
@@ -28,9 +28,9 @@ <h4 class="text-success text-center" translate ng-show="feedCtrl.noFeed">This us
       </a>
     </div>
   </article>
-
-  <article class="feed-list-item" ng-repeat="doc in feedCtrl.documents track by doc.id"
-           ng-init="type = feedCtrl.getDocumentType(doc.document.type);">
+  
+  <article class="feed-list-item flex" ng-repeat="doc in feedCtrl.documents track by doc.id"
+           ng-init="type = feedCtrl.documentType(doc.document.type);">
```

Comment:
If I you use your fonction ... the layout is broken. 
![capture d ecran 2017-03-03 a 20 10 58](https://cloud.githubusercontent.com/assets/3962997/23565359/9a67bada-004d-11e7-8519-5a33150ca5f3.png)

"Feed" css style is not recognized.